### DISCUSSION TITLE
Reset password dead loop if 2FA is not enabled

### DISCUSSION THREAD

**Discussion by MakiseKurisu** [community/community#67919] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

So we all have this friend/colleague: he forgot his password in the time where browser ships password manager with online sync services for free. The excuse this time is that the password was saved but not synced, and then browser data got corrupted. OK, shit happens.

Being this friend, he obviously won't stop there. He also ignores the constant nag from GitHub about mandatory rollout of 2FA. Perfectly in character.

Yet today is the judgement day. 2FA is required to access his account. He complied. However, you need to enter your password to enter sudo mode for setting up 2FA. He then realized he forgot the password and had no way to recover it. No big deal, there is a forgot password link for this exact situation.

However, it redirected to a page asking to enable 2FA before proceeding.

It was at this moment, he knew he fucked up.

At this point, he can't even use the [GitHub support page](https://support.github.com/request) to file a ticket, since that also asks for 2FA to be enabled to proceed. He is entirely locked out of GitHub, with 2 pages he can visit, and both loops back to one another.

Is there any salvation?

**Comment by kuroodo** [comment]
The only salvation is to drop Github for GitLab or anything else to avoid any future crap from Microsoft.

Other than that, have him contact support and see if the person that handles the ticket isn't a prick or is in a decent mood.
